docs/source/migration.rst

```diff
@@ -0,0 +1,471 @@
+|                                               |                                      |
+| ``list_flow_runs``                            |                                      |
++-----------------------------------------------+--------------------------------------+
+| ``flow_action_update``                        | ``FlowsClient.update_run``           |
++-----------------------------------------------+--------------------------------------+
+| ``update_runs``                               | N/A                                  |
++-----------------------------------------------+--------------------------------------+
+| ``flow_action_log``                           | ``FlowsClient.get_run_logs``         |
++-----------------------------------------------+--------------------------------------+
+| ``scope_for_flow``                            | ``SpecificFlowClient.scopes`` [*]_   |
++-----------------------------------------------+--------------------------------------+
+| ``flow_action_resume``                        | ``SpecificFlowClient.resume_run``    |
++-----------------------------------------------+--------------------------------------+
+| ``run_flow``                                  | ``SpecificFlowClient.run_flow``      |
++-----------------------------------------------+--------------------------------------+
+
+
+Converting YAML to JSON
+-----------------------
+
+In the Globus Automate Client, users were allowed to use YAML files to define
+their flows.
+However, the Flows service only accepts JSON data, and YAML was being converted
+to JSON by the client.
+
+Unfortunately, the YAML language specification contains ambiguities, and
+different parsers may treat identical documents differently.
+``globus-cli`` and ``globus-sdk`` do not support YAML parsing, but it is possible
+to convert YAML to JSON using a variety of tools.
+This approach ensures that Globus provided software operates consistently, and
+allows users to continue using YAML or to move off of it, as they prefer.
+
+In this section, we will cover two popular tools for converting YAML to JSON,
+yq (written in Go) and remarshal (written in Python). We will also cover
+one python library, ``pyyaml``, which can be used to load YAML data and pass it
+to the ``globus-sdk``.
+Various other tools provide similar functionality in other languages, and there
+are alternative parsers available in python.
+
+remarshal
+~~~~~~~~~
+
+The `remarshal <https://github.com/remarshal-project/remarshal>`_ project
+provides a wide range of commands for converting data between different
+formats, including YAML and JSON.
+
+These commands exist for the sole purpose of converting data between formats,
+and are therefore a perfect fit for our use-case.
+
+As ``remarshal`` is a python CLI, installation should be performed with
+``pipx``, as with the ``globus-cli``.
+For full instructions, follow `remsarhsal's installation documentation
```

Comment:
```suggestion
For full instructions, follow `remarshal's installation documentation
```

Reply:
I can usually explain my topos as simple innversions of double characacters.
I dddon't have an essay explain nation four remsarhsal.